src/pkg/zoci/common.go

```diff
@@ -25,7 +25,7 @@ const (
 	// ZarfLayerMediaTypeBlob is the media type for all Zarf layers due to the range of possible content
 	ZarfLayerMediaTypeBlob = "application/vnd.zarf.layer.v1.blob"
 	// DefaultConcurrency is the default concurrency used for operations
-	DefaultConcurrency = 3
+	DefaultConcurrency = 6
```

Comment:
This is what the default currently is in the CLI and what is passed throughout the system, but we never updated it here